release/tag_repos.sh

```diff
@@ -147,7 +147,10 @@ create_github_release() {
 	[ -n "${tag}" ] || die "No tag specified"
 	if ! "${hub_bin}" release | grep -q "^${tag}$"; then
 		info "Creating Github release"
-		"${hub_bin}" -C "${repo_dir}" release create -m "${PROJECT} ${tag}" "${tag}"
+		if echo $tag | grep -q -- "-rc"; then
```

Comment:
Looks good, just a hint: bash has support for regex comparison with `[[ "$tag" =~ "-rc" ]]` :)